### ISSUE TITLE
time for a new 3.2.8 release?

### ISSUE THREAD

**Issue by dkg** [jstedfast/gmime#114]
It's been quite a while (and there are cleanup commits in the repo that might be worth applying).  Any chance of pushing out version 3.2.8? or should downstream distributions just merge the outstanding patches?

**Comment by jstedfast** [comment]
Yea, I plan to make a new release soon(ish). I have a local patch to fix issue #112 that I'd like to be able to test first.

**Comment by jstedfast** [comment]
Released 3.2.9 (I guess I bumped to 3.2.8 a year ago but never made an official release?)

https://github.com/jstedfast/gmime/releases/tag/3.2.9

**Comment by dkg** [comment]
Thanks!  https://download.gnome.org/sources/gmime/3.2/ still shows 3.2.7 as the latest release.  That's where the [debian packaging checks for it](https://salsa.debian.org/debian/gmime/-/blob/debian/sid/debian/watch) at the moment. should i just rely on the github tags instead going forward?

**Comment by jstedfast** [comment]
I seem to have lost access to gnome.org (I could probably ask for it back, but in some ways, I'd rather just be self reliant which github allows me to do).

If I figure out how to get gnome.org access back, I'll push the releases there, too, but it's easier to publish on github.

Plus I'd be able to do the gpg signing thing you've asked for in the past.

I don't currently have a gpg key and would need to create one. What would be the recommended way of doing that?

One problem I realized last night was that every gpg key I've created in the past (and subsequently lost) has been for email addresses I no longer have anyway (fejj@helixcode.com/ximian.com/xamarin.com, etc).

I could use my @microsoft.com address or my @gnome.org address, but if I change companies again, my @gnome.org which redirects to my @microsoft.com address goes kaput.

I suppose I don't have to tie it to an email address?

Should I create a key that never expires?

What are the standard practices for this?
